bucket/hadolint.json

```diff
@@ -0,0 +1,21 @@
+{
+    "description": "Dockerfile linter",
+    "homepage": "https://github.com/hadolint/hadolint",
+    "license": "GPL3.0",
```

Comment:
Fixed.

I took the licence from the project page 
![image](https://user-images.githubusercontent.com/7780330/51119934-51a9b600-1814-11e9-9e15-3de36312d5c7.png)

I also looked at the [licence file|https://github.com/hadolint/hadolint/blob/master/LICENSE] itself and it says it's _GNU General Public License v3.0_.

I would like to avoid such mistakes in future. Could you please elaborate on how to determine a licence properly and when (and why) we are supposed to add `*-or-later` postfix?

Reply:
We try to use the license identifiers from https://spdx.org/licenses/, so `scoop info hadolint` would show a link to the license text.